relay-server/src/endpoints/common.rs

```diff
@@ -334,16 +334,15 @@ pub async fn handle_envelope(
     state: &ServiceState,
     envelope: Box<Envelope>,
 ) -> Result<Option<EventId>, BadStoreRequest> {
-    let client_name = envelope
+    let client_name: SdkTag = envelope
         .meta()
         .client_name()
```

Comment:
`client_name` could already return the `SdkTag` (probably would name it just `Sdk` or `SdkName`)

Reply:
IMO mapping a high-cardinality client name to a low cardinality name is purely a concern of the statsd module and a technicality of emitting metrics, it is not an inherent property of `RequestMeta`.